R/data.R

```diff
@@ -31,6 +31,11 @@
 ##'
 ##' @param fit_to_variants Logical, whether to fit to variants data or not
 ##'
+##' @param fit_by_age Logical, whether to fit to age-specific data or not
```

Comment:
Do we want to make this optional or just move to do this all the time? Looks like it is always used in the fits, so leaving the logic here to do one or the other just increases future complexity unless there's a good reason to keep that flexibility

Reply:
Same question applies to `fit_to_under25`

Reply:
True; I've removed this argument as we would only wat to fit by age going forward